app/loggers/log_worker_queue_stats.rb

```diff
@@ -0,0 +1,30 @@
+class LogWorkerQueueStats
+  def self.perform
```

Comment:
`perform` reminds me a lot the workers classes, not totally convinced about the name :D

Reply:
I know, it was `perform`, `run`, or `call`, what do you think? I'm not really picky, I don't really like `call` but couldnt tell you why lol Does `run` sound good?

Reply:
creating names is hard. this class could be named `QueueStatsSidekiqLogWorkerPerformerTracer` :D 

Either is fine, i was just thinking that `perform` reminds me of the `app/workers`